osquery/tables/system/windows/registry.cpp

```diff
@@ -60,10 +60,32 @@ const std::map<DWORD, std::string> kRegistryTypes = {
     {REG_RESOURCE_LIST, "REG_RESOURCE_LIST"},
 };
 
+const char kRegSep = '\\';
+
+void explodeRegistryPath(const std::string& path,
+                         std::string& rHive, 
+                         std::string& rKey) {
+  size_t sepPos = path.find(kRegSep);
+  if (sepPos != std::string::npos) {
+    rHive = path.substr(0, sepPos);
```

Comment:
For future reference :) : https://github.com/facebook/osquery/blob/master/osquery/core/conversions.h#L64